ui/app/src/modules/Preview/PreviewConcierge.tsx

```diff
@@ -240,10 +277,9 @@ export function PreviewConcierge(props: any) {
     let fetchSubscription;
     switch (selectedTool) {
       case 'craftercms.ice.assets':
-        // TODO: aaron to fetch assets here...
+        (assets.isFetching === null && site) && dispatch(fetchAssetsPanelItems(assets.query));
```

Comment:
What I meant was the fetch/load condition needs to consider error. Otherwise, it goes in a loop of fetching. The feedback display comes in the form of ErrorBoundary at the panel level so no need to display the snack.